.docforge/documentation/documentation.yaml

```diff
@@ -59,6 +59,10 @@ structure:
           weight: 20
           persona: Operators
       - fileTree: https://github.com/gardener/gardener/tree/master/docs/concepts
+      - file: gardenadm-large.png
```

Comment:
I think `file: https://github.com/gardener/gardener/blob/master/logo/gardenadm-large.png` should work as there is no renaming. This is relevant on other places as well